front/pages/api/create-new-workspace.ts

```diff
@@ -0,0 +1,67 @@
+import type { WithAPIErrorReponse } from "@dust-tt/types";
```

Comment:
Most of the files seems to be using snake case convention in this folder.

Reply:
We kind of have a mix in `/pages` between snake and kebab. IMHO kebab is the norm for endpoints